pkg/policy/validate_controller.go

```diff
@@ -202,6 +204,14 @@ func (pc *PolicyController) addPolicy(obj interface{}) {
 
 	logger.Info("policy created", "uid", p.UID, "kind", "ClusterPolicy", "name", p.Name)
 
+	// register kyverno_policy_change metric concurrently
+	go func(p *kyverno.ClusterPolicy, policyChangeTimestamp int64) {
+		err := policyChangesMetric.ParsePromMetrics(*pc.promConfig.Metrics).RegisterPolicy(p, policyChangesMetric.PolicyCreated, policyChangeTimestamp)
+		if err != nil {
+			logger.V(4).Error(err, "error occurred while registering kyverno_policy_changes metrics for the above policy's creation", "name", p.Name)
+		}
+	}(p, time.Now().Unix())
```

Comment:
Looks like we register multiple metrics in one place, can we extract it into a function so that we have a clearer view? For example, this can be extracted to `registerPolicyChanges(.., .., logger)`, then we just have one line statement here `go registerPolicyChanges(.., .., logger)`.